ui/src/components/giving/GivingCard.js

```diff
@@ -0,0 +1,208 @@
+import * as React from "react";
+import {
+  Box,
+  Button,
+  Heading,
+  Text,
+  Link,
+  Spacer,
+  useDisclosure,
+  VStack,
+  Flex,
+  Stack,
+  Modal,
+  ModalOverlay,
+  ModalContent,
+  ModalHeader,
+  ModalBody,
+  ModalCloseButton,
+  Center,
+  Image,
+  ModalFooter,
+} from "@chakra-ui/react";
+
+function RenderSwitch(param) {
```

Comment:
Syntax wise, this code seems to be written in a different JS form compared to the rest of our codebase? While not specifically an issue, would be helpful to follow practices that the rest of us are doing so the code is a bit more standardized.